metricflow/test/model/validations/test_validator_helpers.py

```diff
@@ -20,25 +21,76 @@
 
 
 @pytest.fixture
-def base_context_dict() -> ValidationContextJSON:  # noqa: D
+def file_context_dict() -> Dict[str, Union[str, int]]:  # noqa: D
     return {
         "file_name": "foo.py",
         "line_number": 1337,
     }
 
 
 @pytest.fixture
-def base_issue_dict(base_context_dict: ValidationContextJSON) -> ValidationIssueJSON:  # noqa: D
+def data_source_context_dict(file_context_dict: Dict[str, Union[str, int]]) -> Dict[str, Any]:  # noqa: D
```

Comment:
Can you clarify which tests you are talking about? Is this in reference to all the tests in this file in general or perhaps tests that use this fixture? The comment seems to be linked to a fixture for tests instead of a test itself.

Reply:
Sorry, I was referring to use tests that use this fixture.

Reply:
It might be worth some assertions for configuration bits that we want to make sure we get right (like round trip serialization/deserialization on tagged union types, if we end up using those), but otherwise I agree with Paul - we don't need to say "given a valid JSON input we will deserialize it correctly".

Reply:
Sounds good. I think I removed all the relevant tests. Also I believe the [test_jsonifying_and_reloading_model_validation_results_is_equal](https://github.com/transform-data/metricflow/blob/089d9c8a8afbc48666c40d228d532780933d2ee8/metricflow/test/model/validations/test_validator_helpers.py#L123) covers the round trip serialization/deserialization on tagged union types.